lib/tty.js

```diff
@@ -61,7 +60,7 @@ function ReadStream(fd, options) {
   this.isRaw = false;
   this.isTTY = true;
 }
-inherits(ReadStream, net.Socket);
+Object.setPrototypeOf(ReadStream.prototype, net.Socket.prototype);
```

Comment:
Hi, thanks for your contribution first.

We can't only `setPrototypeOf` for `prototype`.

For your information:
> // Change this
util.inherits(A, B);
// To this
Object.setPrototypeOf(A.prototype, B.prototype);
Object.setPrototypeOf(A, B);